test/test_invalid_headers.py

```diff
@@ -113,6 +113,34 @@ def test_transfer_encoding_trailers_is_valid(self, frame_factory):
         request_event = events[0]
         assert request_event.headers == headers
 
+    def test_pseudo_headers_rejected_in_trailer(self, frame_factory):
+        """
+        Ensure we reject pseudo headers included in trailers
+        """
+        headers = self.base_request_headers
```

Comment:
I believe it was because 127 is too long with the full declaration and I felt like this looked a little funky without a param on the first line. I can implement the line wrap though.

```python
header_frame = frame_factory.build_headers_frame(
    self.base_request_headers)
```